packages/SwingSet/README.md

```diff
@@ -116,128 +114,72 @@ The system can pass-by-copy "Data Objects" with similar rules:
 * the object's prototype must be Array or Object (or null)
 * all properties, and the object itself, must be `harden()`ed
 
-## Root Objects
+### Root Objects
 
 The "Root Object" is a callable object returned by `buildRootObject()`. It
 will be made available to the Bootstrap Vat.
 
-## Sending Messages with Presences
+### Sending Messages with Presences
 
 When a Callable Object is sent to another Vat, it arrives as a Presence. This
 is a special (empty) object that represents the Callable Object, and can be
```

Comment:
Seems like we also need to go from `Callable` to `Remotable` ?

Also we may need to clarify that presences are themselves remotable, but they doesn't have local methods.